src/rapids_pre_commit_hooks/lint.py

```diff
@@ -98,7 +98,7 @@ def fix(self):
             key=lambda replacement: replacement.pos,
         )
 
-        for r1, r2 in itertools.pairwise(sorted_replacements):
+        for r1, r2 in more_itertools.pairwise(sorted_replacements):
```

Comment:
I would propose copying the definition from the docs instead of adding a dependency. It’s short.
https://docs.python.org/3/library/itertools.html#itertools.pairwise

```python
def pairwise(iterable):
    # pairwise('ABCDEFG') → AB BC CD DE EF FG
    iterator = iter(iterable)
    a = next(iterator, None)
    for b in iterator:
        yield a, b
        a = b
```

Reply:
I thought about this, but why copy-and-paste code when you can just install a library instead? That's what it's there for. We already have other dependencies in `pyproject.toml`.

Reply:
The other dependencies are somewhat nontrivial. This can be replaced by a 6 line snippet that can be deleted once we drop Python 3.9. Dependencies carry risk (security, API breakage, being unmaintained, etc.).

If we used several of these iterators I would be able to justify it but not for a single function used once.

Reply:
Fair point. I've removed `more-itertools` and used the example from the docs instead.